front/src/Components/CameraControls.svelte

```diff
@@ -78,8 +77,8 @@
     }
 
     let isSilent: boolean;
-    const unsubscribeIsSilent = availabilityStatusStore.subscribe((status) => {
-        isSilent = status === AvailabilityStatus.SILENT;
+    const unsubscribeIsSilent = silentStore.subscribe((silent) => {
```

Comment:
If you want to use `isSilent` with `silentStore` in Svelte, you can do it directly with `$silentStore`

Reply:
I see you use `onDestroy(unsubscribeIsSilent);` why do this?

Reply:
That was an old code. Good catch, it surely can be simplified!